### PR TITLE
Ft 250jewelries

### PR DESCRIPTION
**Pull request by Clarisse-3924**

### CODE REVIEW

File: Benevolencehardware/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Ramadhan0) [inline Benevolencehardware/.vscode/settings.json:1]: please remove **`benevolence hardware`** files

File: Makeupartist/components/about.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<div class="" id="about">
```
Comment (Ramadhan0) [inline Makeupartist/components/about.html:1]: remove **`makeup artist`** files

File: ireme/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Ramadhan0) [inline ireme/.vscode/settings.json:1]: remove **`ireme`** files

File: styledhomeMart/components/about.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<div class="" id="about">
```
Comment (Ramadhan0) [inline styledhomeMart/components/about.html:1]: remove **`style home art`** files

**Review by Ramadhan0 — CHANGES_REQUESTED** [review]

**Comment by fridolinho** [comment]
1. No meta tags
2. No info@ domain email used in contact details and send_email.php

**Pull request closed without merging**